src/index.js

```diff
@@ -0,0 +1,204 @@
+import './style.css';
+import dotsImage from './img/dots.png';
+import syncImage from './img/synchronize.png';
+import enterImage from './img/enter.png';
+import deleteImage from './img/delete.png';
+
+const list = [
+  {
+    description: 'Take a shower',
+    completed: 'false',
+    index: '1',
+  },
+  {
+    description: 'Breakfast',
+    completed: 'false',
+    index: '2',
+  },
+];
```

Comment:
- [ ] Great job so far on implementing all the necessary functionalities mentioned in the project requirements. However, I noticed that a few parts of the projects haven't been fulfilled yet. To completely fulfill all the requirements for this milestone kindly consider the below suggestions:
  - [ ] Kindly adhere to the project requirements that state that all `hardcoded` tasks should be removed from your codebase. 
  - [ ] Consider adhering to the requirements that state a new file should be created to implement all the functionalities of this milestone. See below for more details on the project requirements: 
  
<img width="678" alt="Screenshot 2023-08-18 at 20 25 09" src="https://github.com/eamador1/to-do-list/assets/107192076/5a4fbff8-639b-4795-816b-a77c0de782bc">


Great job once again. Happy coding!😊🚀🍀